src/preset.ts

```diff
@@ -448,13 +449,17 @@ export async function expandConfigurePreset(folder: string,
     expandedPreset.cacheVariables = { };
     for (const cacheVarName in preset.cacheVariables) {
       const cacheVar = preset.cacheVariables[cacheVarName];
-      if (cacheVar && typeof cacheVar !== 'boolean') {
-        if (util.isString(cacheVar)) {
-          expandedPreset.cacheVariables[cacheVarName] = await expandString(cacheVar, expansionOpts);
-        } else if (util.isString(cacheVar.value)) {
-          expandedPreset.cacheVariables[cacheVarName] = { type: cacheVar.type, value: await expandString(cacheVar.value, expansionOpts) };
+      if (cacheVar) {
```

Comment:
If `cacheVar === false` we won't enter this block.  Is that ok?

Reply:
It might be good to also add a `util.isBoolean` helper function.

Reply:
Ah that's a good point